src/ports/postgres/modules/svm/svm.py_in

```diff
@@ -676,27 +985,32 @@ def svm_help(schema_madlib, message, is_svc, **kwargs):
 def svm(schema_madlib, source_table, model_table,
         dependent_varname, independent_varname, kernel_func,
         kernel_params, grouping_col, params, is_svc,
-        verbose, detect_novelty=False, **kwargs):
+        verbose, add_intercept=False, **kwargs):
```

Comment:
Two good points. 
1. I'll rename `add_intercept` to `override_fit_intercept`. That way it's clear that `fit_intercept` is being overwritten when it's true. 
2. Now that I think about it, we don't need to tell the user to use `array[1, ...]` at all, since `fit_intercept` will take care of it both in the case of linear and non-linear SVM. The user can set `fit_intercept=False` if intercept is not desired but default is to always include intercept. 
How does that sound?